net-vpn/protonvpn-cli/protonvpn-cli-2.2.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A VPN command-line tool from protonvpn - python rewrite"
+HOMEPAGE="https://protonvpn.com"
+SRC_URI="https://github.com/ProtonVPN/${PN}-ng/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+KEYWORDS="~amd64"
+SLOT=0
+
+RDEPEND="${PYTHON_DEPS}
+	dev-python/setuptools
```

Comment:
You can use `DISTUTILS_USE_SETUPTOOLS` variable for this one.

Reply:
Actually, I don't see that documented anywhere.  If I make that change, then I receive an error about `BDEPEND`.

Reply:
It is, 
https://devmanual.gentoo.org/eclass-reference/distutils-r1.eclass/index.html
https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/distutils-r1.eclass

the eclass puts setuptools into BDEPEND by default. If you think it's needed in RDEPEND, you specify rdepend for it. If it's not needed at all with your package, you specify "no" for it. I'd trust the automatic detection.